src/codeu/chat/server/MessagePersist.java

```diff
@@ -0,0 +1,182 @@
+
+package codeu.chat.server;
+
+import java.sql.*;
+import java.util.Collection;
+
+import codeu.chat.common.BasicController;
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.RawController;
+import codeu.chat.common.User;
+import codeu.chat.util.Logger;
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+public final class MessagePersist {
+
+	private Connection conn;
+	private Statement stmt;
+
+	public MessagePersist() {
+		try {
+			Class.forName("org.sqlite.JDBC");
+			conn = DriverManager.getConnection("jdbc:sqlite:message.db");
+		}
+		catch (Exception e) {
+			System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+    		System.exit(0);
+		}
+	}
+
+	public void createUserTable() {
+		try {
+			stmt = conn.createStatement();
+			String sql = "CREATE TABLE IF NOT EXISTS USERS " + 

```

Comment:
I like how your SQL queries are formatted nicely and in `ALL_CAPS` (consistent style is important)
However, I'd recommend you change your table names and column names to not be in `ALL_CAPS`. This would make it very clear what is a reserved SQL keyword, and what are your own names
Example of a nicely formatted query:
```
SELECT *
FROM Users
WHERE name = 'Shirene'
```
Style in SQL is usually just a preference, but picking one style and sticking with it makes life easier.